scenarios/centos-9/content_provider.yml

```diff
@@ -9,3 +9,4 @@ cifmw_operator_build_meta_build: true
 cifmw_operator_build_local_registry: 1
 cifmw_operator_build_extra_bundles:
  - "quay.io/openstack-k8s-operators/rabbitmq-cluster-operator-bundle@sha256:9ff91ad3c9ef1797b232fce2f9adf6ede5c3421163bff5b8a2a462c6a2b3a68b"
+cifmw_operator_build_push_registry_tls_verify: false
```

Comment:
I'm pretty sure the CA is injected in the system. Sounds weird we need to do this?

Reply:
OK - content-provider job only.
It's pushing to a local docker registry without and TLS - soo... makes sense, yep!